### ISSUE TITLE
Initializing JS/TS language features not starting

### ISSUE THREAD

**Issue by bvonpotobsky** [microsoft/vscode#219873]
Type: <b>Bug</b>

The JS/TS language service crashed 5 times in the last 5 Minutes.

VS Code version: Code 1.90.2 (Universal) (5437499feb04f7a586f677b155b039bc2b3669eb, 2024-06-18T22:37:41.291Z)
OS version: Darwin x64 23.5.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|9, 9, 10|
|Memory (System)|64.00GB (20.05GB free)|
|Process Argv|. --crash-reporter-id 2a144179-3f3f-4e77-9865-a8da66fed354|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (32)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.2
swagger-viewer|Arj|3.1.2
vscode-tailwindcss|bra|0.12.3
path-intellisense|chr|2.9.0
vscode-eslint|dba|3.0.10
githistory|don|0.6.20
apc-extension|drc|0.3.9
gitlens|eam|15.1.0
moxer-icons|Equ|5.11.1
prettier-vscode|esb|10.4.0
vscode-markdown-todo|fab|2.0.0
auto-rename-tag|for|0.1.10
copilot|Git|1.209.0
copilot-chat|Git|0.16.1
template-string-converter|meg|0.6.1
dotenv|mik|1.0.1
select-part-of-word|mle|1.0.1
vscode-docker|ms-|1.29.1
vscode-language-pack-es|MS-|1.90.2024061209
debugpy|ms-|2024.6.0
python|ms-|2024.8.1
vscode-pylance|ms-|2024.6.1
remote-containers|ms-|0.369.0
vsliveshare|ms-|1.0.5932
color-highlight|nau|2.8.0
vscode-versionlens|pfl|1.14.2
prisma|Pri|5.16.1
vscode-yaml|red|1.15.0
vscode-fileutils|sle|3.10.3
swdc-vscode|sof|2.7.13
errorlens|use|3.20.0
pretty-ts-errors|Yoa|0.5.4

(4 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementsc:30995553
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
jchc7451:31067544
chatpanelt:31048053
dsvsc021:30996838
bdiig495:31013172
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
dwnewjupytercf:31046870
impr_priority:31082723

```

</details>

**Comment by bvonpotobsky** [comment]
Yes, it occurs with all installed extensions disabled and I'm running version 1.90.2 (Universal).
Thanks

**Comment by mjbvz** [comment]
Please try the insiders version too as this has some important fixes

**Comment by bvonpotobsky** [comment]
Same error!

**Comment by cwoolum** [comment]
I've been getting this for a couple months as well in our Monorepos. It completely maxes out our filesystem watchers. I thought it was related to us using Turbo or SSH for our dev environment but it sounds like this affects other users as well.

I've tried disabling all extensions but that didn't help.

**Comment by gaetanlegac** [comment]
I'm having the same issue while opening on a big project. 
Other smaller projects are fine, so it may actually be related to fs watchers.

Version: 1.92.1 (same issue with Nighly)
OS: Darwin arm64 23.1.0
